submariner/pluggable-network-plugin-and-ovn.md

```diff
@@ -0,0 +1,300 @@
+## Add support for OVN with pluggable network-plugin support
+
+[submariner#778](https://github.com/submariner-io/submariner/issues/778)
+
+## Summary
+
+Today Submariner supports working with existing network-plugins which are based
+on the kube-proxy with iptables backend. Any network-plugin outside of this
+case won't work, OVN-Kubernetes is an example.
+
+This is a proposal to support pluggable network drivers, starting with OVN-Kubernetes.
+
+## Proposal
+
+Re-architect Submariner route-agent in a way such that different network-plugin implementations
+can be supported. Those drivers would be notified of different events like:
+
+* `Initialization` event
```

Comment:
![Screenshot from 2020-10-14 11-25-29](https://user-images.githubusercontent.com/6574656/95949329-0e78de80-0e10-11eb-9472-c1a1ebb74a7a.png)

When this is rendered, its displayed as a single sentence and "Called" is capitalized in all the events.

Reply:
Ouch, ok... if it's ok, I'd prefer to merge it as it is... it's only a reference for our development.

Reply:
Ack